### ISSUE TITLE
No indexed suggestions for tags in structure fields

### ISSUE THREAD

**Issue by servicethinker** [getkirby/panel#327]
It's only possible to get suggestions via `data:`. 

(By the way: it's awesome to add tags in structure fields!!)

**Comment by distantnative** [comment]
Seems to work for me in the 2.1 beta. Switching between `index: siblings` and `index: all` changes the scope the tags in my structure field get autocompleted according to either only tags from other siblings' tags field or from all pages' tags field.

**Comment by servicethinker** [comment]
Doesn't work for me at the moment (tested in version ca44895f131611bfbc21fa5c54c9264a30b7e32d). 

Maybe this helps: I have this structure element with a tag field in my blueprint:
```
  cast:
    label: Besetzung
    type: structure
    entry: >
      {{role}}: {{name}}
    fields:
      role:
        label: Rolle
        type: tags
        lowercase: false
        separator: ", "
      name:
        label: Name
        type: tags
        separator: ", "
        lowercase: false
        index: all
```

Nothing helps. I don't get any suggestion for the name field on `index: siblings` and `index: all` - the name is already used in the same field on other pages. Any idea?

**Comment by distantnative** [comment]
I don't know what I did test or why I thought it was working, but testing it with your example it doesn't work for me either.

I think, it is because the autocomplete eventually uses the `pluck()` method, which doesn't really work with structure fields, I believe. `panel/app/controllers/api/helpers.php`at line 56:

```php
$result = $pages->pluck(get('field', 'tags'), get('separator', true), true);
```

**Comment by distantnative** [comment]
@servicethinker I might have found a fix: https://github.com/getkirby/panel/pull/472 - as the description says, I have my doubts that this is a nice fix and if it should actually be implemented. But maybe you might wanna test it and share if this really solves your problem (it seems to work for me, but then again, I thought so earlier and was completely wrong :wink: )

**Comment by distantnative** [comment]
Has been added to the develop branch and will be included in the 2.1 release.

**Comment by servicethinker** [comment]
Works perfect! Thanks!
